test/WebJobs.Script.Tests/Properties/AssemblyInfo.cs

```diff
@@ -39,4 +39,4 @@
 [assembly: AssemblyVersion("1.0.0.0")]
 [assembly: AssemblyFileVersion("1.0.0.0")]
 
-[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly)]
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly, DisableTestParallelization = true)]
```

Comment:
No, I was grasping at straws. I already have parallelization turned off, via CollectionPerAssembly (there is only one assembly). I could probably revert this.